app/views/posts/index.html.erb

```diff
@@ -1 +1,53 @@
-<p>All Posts goes here</p>
+
+
+  <div class="container">
+    <div class="card" style="width: 100%">
+      <ul class="list-group list-group-flush">
+        <li class="list-group-item ">
+            <div class="d-flex justify-content-between">
+              <div class="h5"><b>Name: </b><%=@user.name%></div>
+              <p class="mx-4" style="position: relative">
+                Posts:
+                <span
+                  style="
+                    position: absolute;
+                    width: 20px;
+                    height: 20px;
+                    border-radius: 50%;
+                    background: blue;
+                    color: white;
+                    font-size: 12px;
+                    text-align: center;
+                  "
+                  class=""
```

Comment:
- [ **OPTIONAL** ] For best practices, I think it would be better to avoid the inline styling in your views. Instead, you can use a class and style it in your `app/assets/stylesheets/...` files.